odml/base.py

```diff
@@ -136,11 +135,16 @@ def __contains__(self, key):
             if (hasattr(obj, "name") and obj.name == key) or key == obj:
                 return True
 
-    def append(self, obj):
-        if obj.name in self:
-            raise KeyError("Object with the same name already exists! " + str(obj))
-        else:
-            super(SmartList, self).append(obj)
+    def append(self, *obj_tuple):
+        ll = len(obj_tuple)
+        for i in list(range(0,ll)):
+                if obj_tuple[i].name in self:
+                    raise KeyError("Object with the same name already exists! " + str(obj_tuple[i]))
+                # This check gives ERRORS in validator tests
+                # from odml.section import BaseSection
+                # if not isinstance(obj_tuple[i], BaseSection):
+                #	raise KeyError("Object " + str(obj_tuple[i]) + " is not a Section.")
+                super(SmartList, self).append(obj_tuple[i])
```

Comment:
Why does the commented code raise an error? I would be great to understand the problem